src/tests/SearchResults.test.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
```

Comment:
Tomorrow we'll need to run npm --coverage to check how many more tests we need. I have feeling we may be missing a few